neptune/new/handler.py

```diff
@@ -38,32 +40,49 @@
 from neptune.new.types.value_copy import ValueCopy
 
 if TYPE_CHECKING:
-    from neptune.new.run import Run
+    from neptune.new.attribute_container import AttributeContainer
+
+
+def check_protected_paths(fun):
+    @wraps(fun)
+    def inner_fun(self: "Handler", *args, **kwargs):
+        # pylint: disable=protected-access
+        for path in self.PROTECTED_PATHS:
+            if self._path == path:
+                raise NeptuneProtectedPathException(path)
+        return fun(self, *args, **kwargs)
+
+    return inner_fun
 
 
 class Handler:
-    def __init__(self, run: "Run", path: str):
+    # paths which can't be modified by client directly
+    PROTECTED_PATHS = [
```

Comment:
name it `_PROTECTED_PATHS`? it's an implementation detail for me

Reply:
Ok